sys/fuchsia/fidl_process.txt

```diff
@@ -1,13 +1,90 @@
+	txid		int32[0]
+	reserved	int32[0]
+	flags		int32[0]
+	ordinal		int32[1]
+	infoInLine	fuchsia_process_LaunchInfoInLine
+	infoOutOfLine	fuchsia_process_LaunchInfoOutOfLine
+} [align_8, packed]
 
-resource zx_chan_fuchsia_process_Launcher_client[zx_chan]
-resource zx_chan_fuchsia_process_Launcher_server[zx_chan]
+fuchsia_process_LaunchInfoInLine {
+	executable	flags[fidl_handle_presence, int32]
+	job		flags[fidl_handle_presence, int32]
+	nameInLine	fidl_string
+} [align_8, packed]
 
-zx_channel_create$fuchsia_process_Launcher(options const[0], out0 ptr[out, zx_chan_fuchsia_process_Launcher_client], out1 ptr[out, zx_chan_fuchsia_process_Launcher_server])
+fuchsia_process_LaunchInfoOutOfLine {
+	nameOutOfLine	stringnoz
+} [align_8, packed]
 
-fdio_service_connect$fuchsia_process_Launcher(path ptr[in, string["/svc/fuchsia.process.Launcher"]], handle zx_chan_fuchsia_process_Launcher_server)
+fuchsia_process_LauncherCreateWithoutStartingRequest {
+	txid		int32[0]
+	reserved	int32[0]
+	flags		int32[0]
+	ordinal		int32[2]
+	infoInLine	fuchsia_process_LaunchInfoInLine
+	infoOutOfLine	fuchsia_process_LaunchInfoOutOfLine
+} [align_8, packed]
 
-zx_channel_call$fuchsia_process_LauncherLaunch(handle zx_chan_fuchsia_process_Launcher_client, options const[0], deadline zx_time, args ptr[in, zx_channel_call_args], actual_bytes ptr[out, int32], actual_handles ptr[out, int32], read_status ptr[out, int32])
+fuchsia_process_LauncherAddArgsRequest {
+	txid		int32[0]
+	reserved	int32[0]
+	flags		int32[0]
+	ordinal		int32[10]
+	argsInLine	fidl_vector
+	argsOutOfLine	parallel_array[fidl_string, stringnoz]
+} [align_8, packed]
+
+fuchsia_process_LauncherAddEnvironsRequest {
+	txid			int32[0]
+	reserved		int32[0]
+	flags			int32[0]
+	ordinal			int32[11]
+	environInLine		fidl_vector
+	environOutOfLine	parallel_array[fidl_string, stringnoz]
+} [align_8, packed]
+
+fuchsia_process_LauncherAddNamesRequest {
+	txid		int32[0]
+	reserved	int32[0]
+	flags		int32[0]
+	ordinal		int32[12]
+	namesInLine	fidl_vector
+	namesOutOfLine	parallel_array[fuchsia_process_NameInfoInLine, fuchsia_process_NameInfoOutOfLine]
+} [align_8, packed]
+
+fuchsia_process_NameInfoInLine {
+	pathInLine	fidl_string
+	directory	flags[fidl_handle_presence, int32]
+} [align_8, packed]
+
+fuchsia_process_NameInfoOutOfLine {
+	pathOutOfLine	stringnoz
+} [align_8, packed]
+
+fuchsia_process_LauncherAddHandlesRequest {
+	txid			int32[0]
+	reserved		int32[0]
+	flags			int32[0]
+	ordinal			int32[13]
+	handlesInLine		fidl_vector
+	handlesOutOfLine	array[fuchsia_process_HandleInfoInLine]
+} [align_8, packed]
+
+fuchsia_process_HandleInfoInLine {
+	handle	flags[fidl_handle_presence, int32]
+	id	int32
+} [align_8, packed]
+
+zx_channel_call$fuchsia_process_LauncherLaunch(handle zx_chan, options const[0], args ptr[in, fidl_call_args[fuchsia_process_LauncherLaunchRequest, array[int32], array[int32]]], actual_bytes ptr[out, int32], actual_handles ptr[out, int32], read_status ptr[out, int32])
```

Comment:
Use `array[zx_handle]` for handle arrays.

Reply:
Done.